operations/observability/mixins/workspace/rules/components/nodes/alerts.libsonnet

```diff
@@ -48,6 +48,22 @@
               increase(cluster_autoscaler_failed_scale_ups_total[1m]) != 0
             |||,
           },
+          {
+            alert: 'NetworkConnectionsTooHigh',
+            labels: {
+              severity: 'warning',
+              team: 'workspace'
+            },
+            'for': '10m',
+            annotations: {
+              runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/NetworkConnectionsTooHigh.md',
```

Comment:
@utam0k can you remove the runbook or leave it empty for now? I ask because eventually I think we'll want to reuse [this runbook](https://github.com/gitpod-io/runbooks/pull/371) for high normalized load and high # of connections.

Aside from that, this looks great!

Reply:
Sorry, I couldn't get 🙇 Should I close this PR and make `runbook_url` empty?
https://github.com/gitpod-io/runbooks/pull/379

Reply:
@utam0k this PR is good, no need to close. I was just sharing the runbook doesn't exist, and for the future we can model it after the one I shared.

You are welcome to mark this ready for review, so we can approve. :)